portable-screencap

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
-set -e
-set -o pipefail
```

Comment:
It makes something like `false | true` fail though it succeeds by default:

```
$ false | true  
$ echo $?
0
$ set -o pipefail
$ false | true
$ echo $?
1
```

But as I said in https://github.com/chriscool/ipfs-screencap/commit/fcd55a5ef00ca4ef9f095d95b1f16aac34be3161 it is not portable as dash, which is the default /bin/sh on ubuntu doesn't know about it:

```
$ /bin/sh
$ set -o pipefail
/bin/sh: 1: set: Illegal option -o pipefail
```